src/encoded/commands/deploy.py

```diff
@@ -1024,7 +1024,7 @@ def hostname(value):
     parser.add_argument('--wale-s3-prefix', default='s3://encoded-backups-prod/production-pg11')
 
     # AWS
-    parser.add_argument('--profile-name', default='default', help="AWS creds profile")
+    parser.add_argument('--profile-name', default='default', choices = ("default", "production"), help="AWS creds profile")
```

Comment:
Formatting nit, there shouldn't be spaces around the `=` for kwargs:
```suggestion
    parser.add_argument('--profile-name', default='default', choices=("default", "production"), help="AWS creds profile")
```

Reply:
correct. man, we really need to get some automatic formatting going in this repo.

Reply:
For sure, the current formatting is a mess.